druid/src/widget/split.rs

```diff
@@ -30,6 +30,11 @@ pub struct Split<T> {
     min_bar_area: f64,    // Integers only
     solid: bool,
     draggable: bool,
+    /// The split bar is hovered by the mouse. This state is locked to `true` if the
+    /// widget is active (the bar is being dragged) to avoid cursor and painting jitter
+    /// if the mouse moves faster than the layout and temporarily gets outside of the
+    /// bar area while still being dragged.
+    is_bar_hover: bool,
```

Comment:
Instead of adding a new variable, can we just check `ctx.is_active`?

Reply:
Makes sense I guess. I was porting that from a custom control, didn't realize it's probably not needed. I'll update that PR over the week-end.

Reply:
@cmyr : Actually yes and no (but mostly no from a performance standpoint).

`ctx.is_active()` has already changed when the `MouseMove` event is raised, so we don't know its old value, and therefore cannot know if there is a change between the last and current state. Now, a better way is to move the `set_cursor()` to the `MouseDown` and `MouseUp` in the active case, which doesn't require any variable. And keep the hover check in the non-active case in `MouseMove`. But, in that latter case, if we don't know the previous hover state then we are still forced to be conservative and set the cursor / request a repaint for every single `MouseMove` event. For the price of a single `bool` variable we can avoid that, and I think that's a reasonable price to pay.